modules/build/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -244,8 +253,9 @@ final case class BuildOptions(
     val scalaCliVersion = version
     val launchersTask   = cache.logger.using(task)
 
-    //  If an error occurred while downloading stable versions,
-    //  it uses stable scala versions from Deps.sc
+    /** If an error occurred while downloading stable versions, it uses stable scala versions from
```

Comment:
Not sure this comment was meant to be a scaladoc.

Reply:
It helps in IDEs to leave the comments in this way, so that you can get this information when hovering over the name!